src/sage/schemes/toric/library.py

```diff
@@ -862,25 +790,16 @@ def Cube_face_fan(self, names='z+', base_ring=QQ):
         - ``base_ring`` -- a ring (default: `\QQ`). The base ring for
           the toric variety.
 
-        OUTPUT:
-
-        A :class:`CPR-Fano toric variety
+        OUTPUT: A :class:`CPR-Fano toric variety
         <sage.schemes.toric.fano_variety.CPRFanoToricVariety_field>`.
 
         EXAMPLES::
 
-            sage: Cube_face_fan = toric_varieties.Cube_face_fan()
-            sage: Cube_face_fan
+            sage: Cube_face_fan = toric_varieties.Cube_face_fan(); Cube_face_fan
             3-d CPR-Fano toric variety covered by 6 affine patches
             sage: Cube_face_fan.fan().rays()
-            N( 1,  1,  1),
-            N( 1, -1,  1),
-            N(-1,  1,  1),
-            N(-1, -1,  1),
-            N(-1, -1, -1),
-            N(-1,  1, -1),
-            N( 1, -1, -1),
-            N( 1,  1, -1)
+            N( 1,  1,  1),    N( 1, -1,  1),    N(-1,  1,  1),    N(-1, -1,  1),
+            N(-1, -1, -1),    N(-1,  1, -1),    N( 1, -1, -1),    N( 1,  1, -1)
```

Comment:
This is not necessarily an improvement, as the data in the rectangular format is harder to read than in the vertical format.

But it is okay. I am not suggesting to revert.

Reply:
I could change these from the 2x4 format to a 4x2 format as a compromise?

Reply:
No. Just leave them as they are. I am okay with it.